config/ETH_ddpm.yml

```diff
@@ -0,0 +1,64 @@
+DATA_AGGREGATION:
+  RAW_DATA_DIR: 'datasets/ETH/rawData/'
+  AGG_DATA_DIR: 'datasets/ETH/aggData/'
+PICKLE:
+  PICKLE_DIR: 'datasets/ETH/pickle/'
+  USE_PICKLE: False
+MACROPROPS:
+  # Grid size
+  ROWS: 8
+  COLS: 12
+  # Grid resolution
+  DX: 1.0
+  DY: 1.0
+  LU:
+    - 0
+    - 8
+  # EPSILON value to avoid division by cero
+  EPS: 0.000001
+  OVERLAP: True
+  WINDOWSIZE: 1
+  THETA: 0
+  TIME_RES: 0.4 #Frames for second are 2.5
+DATASET:
+  NAME: "ETH-UCY-4D"
+  SEQ_LEN: 8
+  PAST_LEN: 5
+  FUTURE_LEN: 3
```

Comment:
SEQ_LEN=PAST_LEN+FUTURE_LEN

I suggest to remove SEQ_LEN and define it where is required using PAST_LEN and FUTURE_LEN locally to avoid potential bugs.